otx/algorithms/detection/adapters/mmdet/datasets/tiling.py

```diff
@@ -502,3 +505,76 @@ def get_ann_info(self, idx):
         if "gt_labels" in self.tiles[idx]:
             ann["labels"] = self.tiles[idx]["gt_labels"]
         return ann
+
+    def merge_vectors(self, feature_vectors: List[np.ndarray]) -> List[np.ndarray]:
+        """Merge tile-level feature vectors to image-level feature vector.
+
+        Args:
+            feature_vectors: tile-level feature vectors.
+
+        Returns:
+            merged_vectors (list[np.ndarray]): Merged vectors for each image.
+        """
+
+        # tiles vectors + full image vector
+        vect_per_image = len(feature_vectors) // self.num_images + 1
+        image_vectors = feature_vectors[::vect_per_image]
```

Comment:
Is this intended? If you don't want feature_vectors[:, :vect_per_image], then just resolve this conversation

Reply:
Fixed